starlette_jsonapi/schema.py

```diff
@@ -5,6 +5,7 @@
 from marshmallow_jsonapi import Schema as __Schema, SchemaOpts as __SchemaOpts
 from marshmallow_jsonapi.utils import resolve_params
 from starlette.applications import Starlette
+from starlette_jsonapi.utils import prefix_url_path
```

Comment:
last thing: can you separate the imports here, please? A newline before the addition should do.